deploy/catalog/README.md

```diff
@@ -4,31 +4,77 @@ The Helm Chart deploys Service Catalog into an existing Kubernetes cluster.
 
 ## Prerequisites
 
-  - [Helm](https://github.com/kubernetes/helm) must be installed in the cluster.
-  - Images must be built from source and pushed to a registry accessible to
-    the cluster. Example (from this directory):
-
-    ```
-    $ export REGISTRY=hub.docker.com/<username>
-    $ make -C ../.. push
-    ```
-
-    __Note:__ If deploying locally (e.g. in Minikube), it can be more efficient
-    to use a _local_ Docker registry.
+- [Helm](https://github.com/kubernetes/helm) must be installed in the cluster
+- Images must be built from source and pushed to a registry accessible to the
+  cluster if deploying to a non-local cluster; see the [dev guide](../../docs/DEVGUIDE.md) for instructions
```

Comment:
For _now_... I wouldn't make these changes in wording.

> Images must be built from source and pushed to a registry accessible to the cluster if deploying to a non-local cluster

Since we're not currently publishing our images to any public registry, that sentiment is actually true even if the cluster is local-- the images need to be accessible from a repository that the cluster can reach.

The note, which you're also tentatively removing calls out a meaningful optimization.

All of that being said, when we move past the hacking stage and a `helm install from/local/directory` workflow and this chart becomes "real" and gets published to somewhere like the `kubernetes/charts` repo, there are development-centric details in here (like the assumption that you built the images yourself) that should go away.

So, I'd keep this as it was and we can clean it up more later when we make all the _other_ changes that will also be required to make this a production-quality chart. (There's actually a ton that needs to be done and I will eagerly sign up for that chore.)